packages/components/src/navigation/stories/index.js

```diff
@@ -92,43 +128,55 @@ function Example() {
 					Non-navigation link to Child 2
 				</Button>
 			) : null }
-			<Navigation activeItemId={ active } data={ data } rootTitle="Home">
-				{ ( { level, parentLevel, NavigationBackButton } ) => {
-					return (
-						<>
-							{ parentLevel && (
-								<NavigationBackButton>
-									<Icon icon={ arrowLeft } />
-									{ parentLevel.title }
-								</NavigationBackButton>
-							) }
-							<h1>{ level.title }</h1>
-							<NavigationMenu>
-								{ level.children.map( ( item ) => {
-									return (
-										<NavigationMenuItem
-											{ ...item }
-											key={ item.id }
-											onClick={
-												! item.href
-													? ( selected ) =>
-															setActive(
-																selected.id
-															)
-													: null
-											}
-										/>
-									);
-								} ) }
-							</NavigationMenu>
-						</>
-					);
-				} }
-			</Navigation>
+			<Container>
+				<Navigation
+					activeItemId={ active }
+					data={ data }
+					rootTitle="Home"
+				>
+					{ ( { level, parentLevel, NavigationBackButton } ) => {
+						return (
+							<>
+								{ parentLevel && (
+									<NavigationBackButton>
+										{ parentLevel.title }
+									</NavigationBackButton>
+								) }
+								<NavigationMenu title={ level.title }>
+									{ level.children
+										.filter(
+											( item ) => ! item.isSecondary
+										)
+										.map( ( item ) =>
+											renderMenuItem( item )
+										) }
+								</NavigationMenu>
+								<NavigationMenu
+									title={
+										level.id === 'root'
+											? 'Secondary Menu'
+											: level.title
+									}
+								>
+									{ level.children
+										.filter( ( item ) => item.isSecondary )
+										.map( ( item ) =>
+											renderMenuItem( item )
+										) }
+								</NavigationMenu>
```

Comment:
>The consumer can decide if there is more than one. In the example, I just added isSecondary but this could be a slug or any other property to discern various menus. The title can also be dynamic here- also up to the consumer.

This is true, and this block is a nice example of the consumer having control. What if the entire navigation has 10 of these submenus throughout the hierarchy? Looking at the designs for FSE, its possible that may be the case. You'd need a block like this for every single submenu. `level.children` is getting filtered with each block. I think a better way exists and worth exploring.